libsolidity/codegen/ir/Common.cpp

```diff
@@ -122,3 +127,16 @@ string IRNames::zeroValue(Type const& _type, string const& _variableName)
 {
 	return "zero_" + _type.identifier() + _variableName;
 }
+
+solidity::util::Whiskers& addSourceInformation(solidity::util::Whiskers& _template, ASTNode const& _node, IRGenerationContext const& _context)
+{
+	_template(
```

Comment:
Why does it need to be split into three items in the whiskers template? Couldn't this function just be
```
return "/// @src " + to_string(...) + ":" + to_string(...) + ":" + to_string(...);
````
and then it's used like `t("sourceLocation", sourceLocation(_node, _context))`?

Reply:
Also, maybe this could just be a function of the context?

Reply:
That's a fair point actually. Will change.